resources/wikia/libraries/visit-source/visit-source.js

```diff
@@ -0,0 +1,46 @@
+var VisitSource = (function () {
+	/**
+	 *
+	 * @param cookieName - cookie where session referrer is stored
```

Comment:
Types are available in function signature in source file: https://github.com/Wikia/visit-source/blob/master/src/visit-source.ts#L15